language-plutus-core/src/Language/PlutusCore/TH.hs

```diff
@@ -133,7 +131,7 @@ compileTerm s = do
             quoted :: Quote (Term TyName Name ())
             quoted = do
                 -- See note [Parsing and TH stages]
-                runtimeT <- (fmap discardAnnsTerm . mapInner (Identity . unsafeDropErrors) . parseTerm . strToBs) s
+                runtimeT <- (fmap void . mapInner (Identity . unsafeDropErrors) . parseTerm . strToBs) s
```

Comment:
Oh nice, I hadn't noticed that the fix-ed version was a functor over the annotation. That's much nicer.